panel/command/serve.py

```diff
@@ -406,7 +406,7 @@ def customize_kwargs(self, args, server_kwargs):
                         "base64-encoded bytes."
                     )
                 config.oauth_encryption_key = encryption_key
-            else:
+            elif not config.oauth_encryption_key:
```

Comment:
I agree this fix is indeed correct, but to be clear this fix is simply about the validation right? Because it has no effect on whether or not the env var is respected or not.

Reply:
Yes, I verified that they are being respected - just not verified correctly.

Updated the PR title too.